src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
+    --dry-run \
+    --debug \
+    --set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+    --set global.cluster=<var>K8S_CLUSTER_NAME</var> \
+    --set infrastructure.enabled=true \
+    --set prometheus.enabled=true \
+    --set webhook.enabled=true \
+    --set ksm.enabled=<var>false</var> \
+    --set kubeEvents.enabled=true \
+    --set logging.enabled=true
+    ```
+
+    You can flip the `ksm.enabled` flag to `true` if KSM isn't already present in your cluster.
+
     4. Install the Kubernetes integration:
 
-       ```
-       helm install newrelic/newrelic-infrastructure \
-           --set licenseKey=<var>yYOUR_NEW_RELIC_LICENSE_KEY</var> \ 
-           --set cluster=<var>K8S_CLUSTER_NAME</var> \ 
-           --set config.custom_attributes.cluster=<var>K8S_CLUSTER_NAME</var> \
-           --generate-name
-       ```
+    ```
+    helm upgrade --install newrelic newrelic/nri-bundle \
+    --set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+    --set global.cluster=<var>K8S_CLUSTER_NAME</var> \
+    --set infrastructure.enabled=true \
+    --set prometheus.enabled=true \
+    --set webhook.enabled=true \
+    --set ksm.enabled=<var>false</var> \
+    --set kubeEvents.enabled=true \
+    --set logging.enabled=true
+    ```
+
     5. Wait a few seconds, then check that the `DaemonSet` and pods have been created:
 
-       ```
-       kubectl get daemonsets,pods
-       ```
+    ```
+    kubectl get daemonsets,pods
+    ```
+
     6. Make sure you see a `DaemonSet`, and one pod per node.
+
+    ### Configuring `nri-bundle` with Helm
+
+    #### Enabling and disabling subcharts
+
+    The `nri-bundle` chart pulls the following dependencies, if the corresponding flags are enabled:
+    
+
+    | Subchart | Flag |
+    |----------|------|
+    | `newrelic-infrastructure` | `infrastructure.enabled` |
+    | `nri-prometheus` | `prometheus.enabled` |
+    | `nri-metadata-injection` | `webhook.enabled` |
+    | `kube-state-metrics` | `ksm.enabled` |
+    | `nri-kube-events` | `kubeEvents.enabled` |
+    | `newrelic-logging` | `logging.enabled` |
+
+    #### Configuring subcharts
+
+    The `nri-bundle` chart leverages [Helm's subchart configuration
+    mechanism](https://helm.sh/docs/chart_template_guide/subcharts_and_globals) to configure its components. This means
+    that you can supply the configuration values to the individual components of the chart, as listed at the beginning of this section, by prefixing them with the name of the chart. 
+    
+    For example, to run the agent in
+    unprivileged mode, you can add `--set newrelic-infrastructure.privileged=false` to the command above, where
+    `newrelic-infrastructure` is the name of the sub-chart (as listed at the beginning of the section), and `privileged`
+    is the configuration value for said chart.
+
+    Please refer to [each of the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/) to see all
```

Comment:
I would add to check the values.yaml to see some examples

Reply:
✔️